components/auth/auth-form.test.tsx

```
@@ -1,544 +1,587 @@
+        expect(
+          screen.getByText(/password must be at least/i),
+        ).toBeInTheDocument();
+      });
+    });
 
-    it('handles extremely long input values gracefully', async () => {
-      const longEmail = 'a'.repeat(500) + '@example.com'
-      const longPassword = 'p'.repeat(1000)
-      render(<AuthForm mode="login" onSubmit={mockOnSubmit} />)
+    it("handles extremely long input values gracefully", async () => {
+      const longEmail = "a".repeat(500) + "@example.com";
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)